docs/global-styles.md

```diff
@@ -0,0 +1,20 @@
+# Global Styles
+
+In GOV.UK Frontend we want to provide a convenient way to have correct styles applied to your links (`<a>`) and paragraphs (`<p>`) without having to set required classes for each instance of an element.
+
+We have chosen 'Opt-in' as the default behaviour.
```

Comment:
Shall we say 'Opted-in'. For me 'Opt-in' sound like I have the option to choose if I want it, which is the opposite of what we're doing by default. We can ask @amyhupe to help with wording.